main.py

```diff
@@ -280,14 +280,16 @@ def hello():
     return 'Hello, Circles!!'
 
 
-@app.route('/getid')
+@app.route('/circles/api/v1.0/getid')
 def getid():
     # Fetch the person making the request
-    req_token = request.headers.get('Authorization')
+    req_token = request.headers.get('Authorization').lower()
     req_user = (Person.match(graph).where("_.email = '{}'".format(req_token))).first()
     return str(req_user.__primaryvalue__)
 
 
+
```

Comment:
Generally you do two new lines between "sections" of code, so delete these please!